ui-tests/src/test/java/org/openmrs/reference/VisitNoteTest.java

```diff
@@ -26,20 +25,20 @@
     private static final String DIAGNOSIS_PRIMARY = "CANCER";
     private static final String DIAGNOSIS_SECONDARY = "MALARIA";
     private static final String DIAGNOSIS_SECONDARY_UPDATED = "Pneumonia";
-
+    private PatientVisitsDashboardPage patientVisitsDashboardPage;
```

Comment:
There are trailing spaces before private too, you can get rid of those by pressing the cursor on the top line, press enter and immediately start typing vs using the cursor at the start of the line and using spacing to fix indention spacing

Reply:
Resolved thanks